### PR TITLE
[stable/redis] Add nodeSelector and tolerations

### PR DESCRIPTION
**Pull request by cburmeister**
This commit adds support for `nodeSelector` and `tolerations` to the
`stable/redis` chart.

### CODE REVIEW

File: stable/redis/README.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ The following tables lists the configurable parameters of the Redis chart and th
 | `metrics.imageTag`         | Exporter image                        | `v0.11`                                                   |
 | `metrics.imagePullPolicy`  | Exporter image pull policy            | `IfNotPresent`                                            |
 | `metrics.resources`        | Exporter resource requests/limit      | Memory: `256Mi`, CPU: `100m`                              |
+| `nodeSelector`             | Node labels for pod assignment        | {}                                                        |
+| `tolerations`              | Toleration labels for pod assignment  | {}                                                        |
```
Comment (unguiculus) [inline stable/redis/README.md]: `{}` -> `[]`.

File: stable/redis/values.yaml [hunk 2]
```diff
@@ -52,3 +52,9 @@ resources:
   requests:
     memory: 256Mi
     cpu: 100m
+
+## Node labels and tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
+nodeSelector: {}
+tolerations: {}
```
Comment (unguiculus) [inline stable/redis/values.yaml]: `tolerations: []`

Should be a list.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by cburmeister** [comment]
thanks @unguiculus !

**Comment by unguiculus** [comment]
@k8s-bot ok to test

**Review by unguiculus — APPROVED** [review]

**Pull request merged**